e2e/cypress/integration/custom_status/custom_status_spec.js

```diff
@@ -0,0 +1,557 @@
+        });
+
+        it('MM-T3836_5 "In a meeting" is selected with the calendar emoji', () => {
+            // * Default emoji is currently visible in the custom status input
+            cy.get('#custom_status_modal .StatusModal__emoji-button span').should('have.class', 'icon--emoji');
+
+            // * Input should be empty
+            cy.get('#custom_status_modal input.form-control').should('have.value', '');
+
+            // # Select a custom status from the suggestions
+            cy.get('#custom_status_modal .statusSuggestion__content').contains('span', customStatus.text).click();
+
+            // * Emoji in the custom status input should be changed
+            cy.get('#custom_status_modal .StatusModal__emoji-button span').invoke('attr', 'data-emoticon').should('contain', customStatus.emoji);
+
+            // * Selected custom status text should be in the input
+            cy.get('#custom_status_modal input.form-control').should('have.value', customStatus.text);
+        });
+
+        it('MM-T3836_6 should set custom status when click on Set Status', () => {
+            // # Click on the Set Status button
+            cy.get('#custom_status_modal .GenericModal__button.confirm').click();
+
+            // * Modal should be closed
+            cy.get('#custom_status_modal').should('not.exist');
+
+            // * Status should be set and the emoji should be visible in the sidebar header
+            cy.get('#headerInfoContent span.emoticon').invoke('attr', 'data-emoticon').should('contain', customStatus.emoji);
+        });
+
+        it('MM-T3836_7 should display the custom status tooltip when hover on the emoji in LHS header', () => {
+            // # Hover on the custom status emoji in the sidebar header
+            cy.get('#headerInfoContent span.emoticon').trigger('mouseover');
+
+            // * Custom status tooltip should be visible
+            cy.get('#custom-status-tooltip').should('exist');
+
+            // * Tooltip should contain the correct custom status emoji
+            cy.get('#custom-status-tooltip .custom-status span.emoticon').invoke('attr', 'data-emoticon').should('contain', customStatus.emoji);
+
+            // * Tooltip should contain the correct custom status text
+            cy.get('#custom-status-tooltip .custom-status span.custom-status-text').should('have.text', customStatus.text);
+        });
+
+        it('MM-T3836_8 should open custom status modal when emoji in LHS header is clicked', () => {
+            // * Check that the custom status modal is not open
+            cy.get('#custom_status_modal').should('not.exist');
+
+            // # Click on the custom status emoji in the sidebar header
+            cy.get('#headerInfoContent span.emoticon').click();
+
+            // * Check that the custom status modal should be open
+            cy.get('#custom_status_modal').should('exist');
+        });
+    });
+
+    describe('MM-T3846 Setting your own custom status', () => {
```

Comment:
Please move to its own file `custom_status_3_spec.js`

```suggestion
    describe('Custom Status - Setting Your Own Custom Status', () => {
```